### ISSUE TITLE
'May' and 'might' are erroneously defined

### ISSUE THREAD

**Issue by pivic** [MicrosoftDocs/microsoft-style-guide#187]
I just read this section in 'Can, may':

> Use might to express possibility. Don't use may, which might be interpreted as providing permission.

From Merriam-Webster on '[may](https://www.merriam-webster.com/dictionary/may)':

> —used to indicate possibility or probability

From Merriam-Webster on '[might](https://www.merriam-webster.com/dictionary/might)':

> past tense of MAY

When it comes to grammar, Microsoft refer to Chicago Style: https://docs.microsoft.com/en-us/style-guide/grammar/grammar-and-parts-of-speech

Check out what they say and you'll find that they disagree with your definition.

American Heritage—which you also refer to at times—could be wrangled where it comes to one of the terms, but I'd rather go with the mainly-known definitions from Merriam-Webster and Chicago Style.

In other words, your section—quoted at the top of this GitHub issue—is erroneous, especially in the American English sense of the words.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d785974e-5799-3b8f-8dc5-4c2e1d285aa6
* Version Independent ID: 999d68ed-14be-06ba-afd1-0c3f45686acd
* Content: [can, may - Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/can-may)
* Content Source: [styleguide/a-z-word-list-term-collections/c/can-may.md](https://github.com/MicrosoftDocs/microsoft-style-guide/blob/master/styleguide/a-z-word-list-term-collections/c/can-may.md)
* Product: **non-product-specific**
* GitHub Login: @pallep
* Microsoft Alias: **pallep**
